resources/build.js

```diff
@@ -0,0 +1,45 @@
+var execSync = require('child_process').execSync;
+var fs = require('fs');
+var rimraf = require('rimraf');
+var isWindows = /^win/.test(process.platform)
+
+var out = function (command) {
+  execSync(command, {stdio:'inherit'});
+};
+
+console.log('"Running node ./resources/build.js"');
+
+if (!isWindows) {
+  out('set -e'); // exit on console error
+} // put windows equivalent here if you know it
+
+if (!fs.existsSync('node_modules/.bin')) {
+  console.log('"Be sure to run \`npm install\` before building GraphiQL."');
+}
+
+var distDir = './dist';
+rimraf.sync(distDir);
+fs.mkdirSync(distDir);
+
+out('babel src --ignore __tests__ --out-dir dist/');
+console.log('"Bundling graphiql.js..."');
+out('browserify -g browserify-shim -s GraphiQL dist/index.js > graphiql.js');
+console.log('"Bundling graphiql.min.js..."');
+if (isWindows) {
+  out('browserify -g browserify-shim -g uglifyify -s GraphiQL dist/index.js | uglifyjs -c --screw-ie8 > graphiql.min.js');
+} else {
+  out('browserify -g browserify-shim -g uglifyify -s GraphiQL dist/index.js 2> /dev/null | uglifyjs -c --screw-ie8 > graphiql.min.js 2> /dev/null');
```

Comment:
I think we can do this without piping to the shell but using real pipes in a platform-agnostic way, so we won't need the `isWindows` check here.